lib/bundler/stats/tree.rb

```diff
@@ -21,12 +21,22 @@ def summarize(target)
     }
   end
 
+  def version_requirements(target)
+    transitive_dependencies = transitive_dependencies(target, requirement: true)
+    { name: target,
+      total_dependencies: transitive_dependencies.count,
+      first_level_dependencies: first_level_dependencies(target).count,
+      top_level_dependencies: reverse_dependencies_with_versions(target),
+      transitive_dependencies: transitive_dependencies,
+    }
+  end
+
   def first_level_dependencies(target)
     raise ArgumentError, "Unknown gem #{target}" unless @tree.has_key? target
     @tree[target].dependencies
   end
 
-  def transitive_dependencies(target)
+  def transitive_dependencies(target, requirement: false)
```

Comment:
I think this parameter is unused. Am I missing it somewhere?

Reply:
True, it's a relic. I'll fix it.